repos/system_upgrade/common/actors/redhatsignedrpmscanner/actor.py

```diff
@@ -55,7 +55,7 @@ def has_katello_prefix(pkg):
             return pkg.name.startswith('katello-ca-consumer')
 
         upg_path = rhui.get_upg_path()
-        whitelisted_cloud_flavours = ('azure', 'azure-sap', 'google', 'google-sap')
+        whitelisted_cloud_flavours = ('azure', 'azure-eus', 'azure-sap', 'google', 'google-sap')
```

Comment:
We may add a comment that "AWS" is missing here because those packages are signed by RH.

Reply:
Fixed